packages/codemod/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
I think we can extend one of the existing tsconfigs, maybe [this one](https://github.com/edgedb/edgedb-js/blob/master/packages/tsconfig/base.json), like in https://github.com/edgedb/edgedb-js/blob/df2faf9e4bcdfade1837f2efab0e8a61cdaa83f9/packages/driver/tsconfig.cli.json#L2